modules/generic_worker/templates/generic-worker.plist.erb

```diff
@@ -12,7 +12,7 @@
         <string>/usr/local/bin/run-generic-worker.sh</string>
         <string>run</string>
         <string>--config</string>
-        <string>/etc/generic-worker.config</string>
+        <string>/etc/generic-worker/config</string>
```

Comment:
If you are going put the config in `/etc/generic-worker/config`, you should also create the dir `/etc/generic-worker/`